### ISSUE TITLE
Unknown place of origin for a manuscript

### ISSUE THREAD

**Issue by atoboy** [djbpitt/repertorium#10]
In some descriptions we have:
```xml
<origPlace>Unknown</origPlace>
```
In most of the descriptions `<origPlace>` is missing, which means it is _Unknown_. 
Are we going to add 
```xml
<origPlace>Unknown</origPlace>
```
in the descriptions, where this information is not explicitly encoded, or we are going to remove the element along with its content, and keep it only where we have some idea about the place of origin?

**Comment by djbpitt** [comment]
@atoboy I agree that we should get rid of the inconsistency, so either 1) we should always specify unknown when the place of origin is unknown (at least to the person preparing the description) or 2) we never specify it if the place is unknown, and specify it only if we know of a specific place. 

In principle these two strategies can be informationally identical, except that if we make the `<origPlace>` element optional, we could accidentally forget to enter a known place and the system will conclude that the place is unknown. If we always require `<origPlace>`, even when the value is "unknown", we reduce the likelihood of our making a mistake because we'll have to enter something specific, so we can't just forget that the element exists.

I think this is part of a more general question: should default values be implicit or explicit? If we make `<origPlace>` optional, we make "unknown" an implicit default. If we make it required, the default must be recorded explicitly. At an extreme, if there is no description of, say, binding or decoration, does that mean that there is no binding or decoration, or does it mean that we don't have access to information about the binding or decoration, or does it mean that we chose not to record information about the binding or decoration because it wasn't interesting for our purposes? In those cases I think the default assumption should probably be that there is no default: the absence of a binding or decoration description element does not mean that there is no binding or decoration, although that might be the case. 

I don't have a strong opinion about which approach is best in the case of `<origPlace>`, and the point of my introducing binding and decoration here is that I think we probably should make an explicit decision separately about each situation, as we notice them, where we are considering making an element optional and we then have to document what its omission means. With respect to an unknown place of origin, what would you recommend: enter `<origPlace>` only when the value is known or make it required and specify "unknown" for unknown values?

**Comment by atoboy** [comment]
@djbpitt
I would simply not enter `<origPlace>` when we have not enough data. When we make an index of places of origin than XQuery will not find this element and will display the information just when `<origPlace` contains some text. Most of the descriptions do not contain this element when the encoder does not know the place of origin. So, it will easier in these cases just to remove the element and its content: `<origPlace>unknown</origPlace>`.

**Comment by djbpitt** [comment]
@atoboy No object here! I'll move this card into the **To be implemented** column.
